feat(content): Bringing a lost dog to its owner on Glory (#9961)
endless-sky/endless-sky: data/human/human missions.txt

			`	As you approach the exit to the park a small dog pads over to you and looks at you with mournful eyes. It seems to be alone, with no owner anywhere in sight.`
			choice
				`	(Call to the dog.)`
					goto doglover
				`	(Ignore the dog.)`

			`	You ignore the dog and eventually it sulks away to interact with another patron of the park. As you return to your ship it begins to get dark, and you feel your day of quiet contemplation come to a close.`
				decline

			label doglover
			`	"Come here," you call to the dog, which looks to be a poodle mix, and it obediently does as you command. You see that the dog is a male and check his collar:`
			``
			`	"Caesar Augustus - Property of the Mitchells"`
			`	3 Mitchell Drive`
			`	Palmyra City, Avalonia District,`
			`	Glory, Wazn System`
			``
			`	"That's odd," you say to Caesar. "How did you get lost on the wrong planet, huh boy?"`
			choice
				`	(Take Caesar to his owners on Glory.)`
					goto takehim
				`	(Leave Caesar here.)`

> 			`	Perhaps his owners are on vacation and will be back looking for him any minute. Although it's getting dark and no one seems to be around, you decide the best course of action is to leave Caesar where you found him.`

Reviewer: ```suggestion
			action
				event "lost dog: homeward bound" 90 180
			`	Perhaps his owners are on vacation and will be back looking for him any minute. Although it's getting dark and no one seems to be around, you decide the best course of action is to leave Caesar where you found him.`
```

			`	The butler arrives and looks you over unhappily. "The lady of the manor is not expecting... guests. Perhaps, you've misread your appointment, or more likely, arrived at the wrong estate."`
			choice
				`	(Show Caesar to the butler and ask to see his owner.)`

			`	The butler's frown deepens as you show Caesar to him.`
			`	He sighs and says, "Card, please." He holds out a silver tray and awaits your calling card.`
			choice
				`	(Explain you don't have a calling card.)`
					goto nocard
				`	(Place your card on the tray.)`
					to display
						"net worth" > 20000000

			`	The butler's eyebrows rise in the slightest hint of surprise when you place your card on the tray. He promptly turns and disappears into the overlarge house. You, Caesar, and the footman all wait for half an hour before a woman arrives at the front entrance, wearing an exceedingly complicated gown.`
				goto boolinbrook

			label nocard

> 			`	The butler smiles smugly. "How unfortunate. In that case, I have no choice but to bring you in through the servant's entrance. It's only proper."`

Reviewer 2: ```suggestion
			`	The butler smiles smugly. "How unfortunate. In that case, I have no choice but to bring you in through the servants' entrance. It's only proper."`
```

			`	She turns her attention back to the dog and speaks to him in baby talk. "But, instead, this gallant captain saved you!"`
			`	The lady snaps her fingers, and the butler is immediately at her side. "We must make this person whole. I can't be perceived as owing favors to vagabonds. Imagine the talk!`
			`	The butler quickly procures a credit chip and shows it to her, accepting a curt nod for approval. "I realize that this is probably a fortune to you," she says. "I recommend you find a financial advisor and invest it well."`
			`	With that, she turns and leaves; the butler likewise disappears, and you're escorted by the footman back to your vehicle.`
			`	You look down at the credit chip in your hand and read its value: <payment>.`

mission "Lost Dog: Homeward Bound"
	to offer
		random < 3

> 		has "Lost Dog: declined"

Reviewer 2: I haven't tested this yet, but it seems this mission would trigger even if I never went to the park?

> 		random < 3
> 		has "Lost Dog: declined"

Reviewer: ```suggestion
		has "event: lost dog: homeward bound"
```

			`As you are walking around the spaceport, you see that more than a few pilots are watching the news on nearby displays. There are mixed emotions amongst the crowd - some pilots are having a good laugh while others seem disappointed or even confused. You aren't sure what this is about until, lo and behold, the result of that Hogshead naming poll from a while ago is clearly shown on a display - Shippy McShipface. You can't say it's all that much worse than some of the names you've heard pirates give their ships before, but it still seems silly that the transport company is actually going through with the name.`
			`	All of the hyped discussions around Hogsheads in the past gave them a brief but strong popularity amongst transport organizations looking to grab the average passenger's attention. In the end, the company responsible for this naming poll got what they wanted, and a little bit of the Hogshead's spirit has been brought back, along with the interest of several pilots. Perhaps the name "Shippy McShipface" is fitting after all.`
				decline

mission "Lost Dog"

> 	name "Bring a dog to <planet>"

Reviewer 3: ```suggestion
	name "Bring a dog to <destination>"
```

Reviewer: We always use `<planet>` instead of `<destination>` in mission names.

Reviewer 3: True. Sorry, I thought this was the mission's description.

Reviewer: No problem.

> 	name "Bring a dog to <planet>"

Reviewer 4: ```suggestion
	name "Lost dog to <planet>"
```

Author: I don't understand the change? Doesn't that make less sense?

> 	description "Bring Caesar, a dog you found lost on <source>, to their owner on <destination>."

Reviewer 4: With "you found lost," having two verbs like that makes it a bit confusing.

Reviewer 3: ```suggestion
        description "Bring Caesar, a dog lost on <source>, to their owners on <destination>."
```

Reviewer: ```suggestion
	description "Bring Caesar, a dog you found on <source>, to their owners on <destination>."
```
Perhaps better?

	to offer
		random < 15
	source "Calda"

> 	destination "Hestia"

Reviewer 4: "Pookie, Part 2" is also a mission involving a dog which you're returning to its owner on Hestia. Maybe this one should involve some other destination planet?

Author: The mission works as a way to give the player a glimpse into what life is like on Hestia. A brief window into the life of the super-rich who are said to live there. 
That is to say the main character of this mission is Hestia (and to a lesser degree its relationship with Calda). To move it to a different world would be essentially require writing a brand new mission from the ground up.

Reviewer 4: Seems to me like you could set it on Martini instead without having to do any major changes, if it occurred before the bombings. Or Farseer. Or many of the other very rich paradise planets.

Author: I like the idea of a Martini mission, and I appreciate your point. This was written at a time before the other dog mission existed - hence why it's happened like this. But I still feel like it's too nested within the specific atmosphere of Hestia to be moved without a significant re-write, and I am not prepared to do that.

Reviewer 5: I don't think that Hestia is particularly unique in having a clique of ultra-rich people. I'm also not sure how this mission is inextricable from being on Hestia; I could easily see the mission's destination being Glory, for instance. In my opinion, changing the origin of this mission doesn't actually require any significant rewrites.

Author: I won't stop you or anyone else if they want to make that change.

	on offer
		conversation

> 			`Your walk from the <ship> takes you outside <planet>'s bustling spaceport and into the overtly welcoming and carefully inoffensive tourist-saturated streets of the spaceport town. Every path seems purpose-built to whisk you further away from your ship and deeper into the unnatural radiance of this Paradise World. The hand-crafted beauty is inescapable, a never-ending sequence of breathtaking views.`

Reviewer 4: This seems to be right on the line between good storytelling and telling me what I think. Also, three hyphens in one paragraph is a lot.

Author: As long is walks the line, right?

Reviewer 4: Right, although it could probably be tweaked slightly to make it read better.

Author: If you have some suggestions? I've re-read it and I feel like it's not telling you what to think. This is a planet that has been built to be 'objectively beautiful' - something you are noticing. The player can decide how they feel about that though.

> 			`	As you walk you eventually come across an immaculate and perfectly manicured garden park. Normally there is an exorbitant fee to enter, but due to some promotion today you can enter freely.`

Reviewer 4: How would I know of the fee to enter if I'm just wandering about outside the spaceport and have never been to the park before?

> 			`	As you walk you eventually come across an immaculate and perfectly manicured garden park named "Parc Manifold." You walk towards the gates only to discover a ticket office, with signs advertizing the price of entry as a mere 3,400 credits. But due to some promotion, tickets are free for the week.`

Reviewer 3: Just:
* you made a misspelling at "advertising", it's advertizing.
* I think credit should be plural

> 			`	As you walk you eventually come across an immaculate and perfectly manicured garden park named "Parc Manifold." You walk towards the gates only to discover a ticket office, with signs advertizing the price of entry as a mere 3,400 credits. But due to some promotion, tickets are free for the week.`

Author: > Just:
> 
>     * I think you made a misspelling at "advertising", it should be advertizing, as advertize

In American English its advertising.

> 			`	As you walk you eventually come across an immaculate and perfectly manicured garden park named "Parc Manifold." You walk towards the gates only to discover a ticket office, with signs advertizing the price of entry as a mere 3,400 credits. But due to some promotion, tickets are free for the week.`

Reviewer 6: @SpearDane is right. The correct spelling is `advertising`, from `advertise`. `3,400 credit regular admission fee` is correct as well. In the latter case, `credit` is acting as a description for `fee`. There's no need for a plural.

> 			`	As you walk you eventually come across an immaculate and perfectly manicured garden park named "Parc Manifold." You walk towards the gates only to discover a ticket office, with signs advertizing the price of entry as a mere 3,400 credits. But due to some promotion, tickets are free for the week.`

Reviewer 3: My bad, but I'm pretty sure it's `advertize` and not `advertise`

> 			`	As you walk you eventually come across an immaculate and perfectly manicured garden park named "Parc Manifold." You walk towards the gates only to discover a ticket office, with signs advertizing the price of entry as a mere 3,400 credits. But due to some promotion, tickets are free for the week.`

Author: Guy, it's not. Like, it really really is not. 

https://en.wiktionary.org/wiki/advertize
https://en.wiktionary.org/wiki/advertise#English

You'll notice the spelling with a 'z' is listed as "archaic" for the US. 

That is not the regular spelling for advertise in the United States of America.

> 			`	As you walk you eventually come across an immaculate and perfectly manicured garden park named "Parc Manifold." You walk towards the gates only to discover a ticket office, with signs advertizing the price of entry as a mere 3,400 credits. But due to some promotion, tickets are free for the week.`

Reviewer 3: "The z spelling is an archaic form in British English", " It appears occasionally in modern American English"

> 			`	As you walk you eventually come across an immaculate and perfectly manicured garden park named "Parc Manifold." You walk towards the gates only to discover a ticket office, with signs advertizing the price of entry as a mere 3,400 credits. But due to some promotion, tickets are free for the week.`

Reviewer 3: archaic for the UK, but regular for the US

> 			`	As you walk you eventually come across an immaculate and perfectly manicured garden park named "Parc Manifold." You walk towards the gates only to discover a ticket office, with signs advertizing the price of entry as a mere 3,400 credits. But due to some promotion, tickets are free for the week.`

Author: Exactly. It's non-standard in American English.

> 			`	As you walk you eventually come across an immaculate and perfectly manicured garden park named "Parc Manifold." You walk towards the gates only to discover a ticket office, with signs advertizing the price of entry as a mere 3,400 credits. But due to some promotion, tickets are free for the week.`

Reviewer: We exclusively use "advertise" in Endless Sky.

> 			`	As you walk you eventually come across an immaculate and perfectly manicured garden park named "Parc Manifold." You walk towards the gates only to discover a ticket office, with signs advertising the price of entry being a mere 3,400 credits. But due to some promotion, tickets are free for this week.`

Reviewer 4: Advertising/being is a bit repetitive in structure, and that makes it a little confusing. How about:
```suggestion
			`	As you walk you eventually come across an immaculate and perfectly manicured garden park named "Parc Manifold." You walk towards the gates only to discover a ticket office, with signs advertising the price of entry as a mere 3,400 credits. But due to some promotion, tickets are free for this week.`
```

Reviewer 3: ```suggestion
		`	As you walk you eventually come across an immaculate and perfectly manicured garden park named "Parc Manifold." You walk towards the gates only to discover a ticket office, with signs advertizing the price of entry as a mere 3,400 credits. But due to some promotion, tickets are free for the week.`
```

			choice
				`	(Go to the park.)`
					goto park
				`	(Head back to your ship.)`

> 			`	It was a splendid walk, but you decide it's time to head back. You have more important things to do than wander around an expensive terraformed garden.`

Reviewer 4: Again, "it was a splendid walk" and "you have more important things to do" imply that I'm (a) not just a tourist exploring this planet while also getting close to telling me what I think. This sounds like a sentence out of a game of interactive fiction that is trying to hint towards a specific course of action.

				decline

> 			label park

Reviewer 4: I feel like this section still needs a bit of work. It could use a little bit of fleshing-out, maybe an option on whether or not to approach the dog, and maybe a description of what makes the scene beautiful rather than a description of my attitude toward it.

			`	The park has been meticulously sculpted so that no matter where you stand it appears as though you are in the center of a lush naturalist painting. Colorful, comforting flora bleed away from you at all angles, and adorable yet inaccessible fauna seem to hug the very periphery of your every step.`

> 			`	The air is clear and crisp, and even the crunch and thud of the ground beneath your feet seem somehow delicious and satisfying. Finally, the sun begins and set and it is time to head back.`

Reviewer 4: What if I don't like taking walks in nature?

Author: Then why did you go on one?

			`	As you approach the exit to the park a small dog pads over to you and looks at you with morunful eyes. The dog seems to be alone with no owner anywhere in sight.`

> 			`	"Come here," you call to the poodle-hybrid, who obediently does as you command. You see that it is a male and check its collar:`

Author: _Sigh_. 
You're right.

			``
			`	"Caesar Augustus - Property of the Mitchells"`
			`	3 Mitchell Drive`
			`	Avalonia District, Hestia`
			``
			`	"That's odd," you say to Caesar. "How did you get lost on the wrong planet, huh boy?"`
			`	You pet Caesar, who seems glad of the attention, and decide what to do.`
			choice
				`	(Take Caesar to his owners on Hestia.)`
					goto takehim
				`	(Leave Caesar here.)`

			`	Perhaps his owners are on vacation and will be back looking for him any minute. Although it's getting dark and no one seems to be around, you decide the best course of action to leave Caesar at the park.`

> 			`	As you walk off you hear Caesar whining, and as you look back you notice him slink off into the park. But you've made your decision; hopefully everything will work out for the best.`

Reviewer 4: Again, the as you/as you is a bit repetitive, and the last sentence is probably unnecessary.

```suggestion
			`	As you turn away, Caesar begins to whine, but soon quiets down when you don't come back to pick him up. Looking back, you notice him slink off into the shadow of a perfectly manicured shrub.`
```

				decline

			label takehim

> 			`	"Well, come on boy. Let's go." Caesar eagerly follows you back to your ship. He seems greatly relieved to be with you, and not left all alone at the park.`

Reviewer 4: Written like this, it looks like Caesar is the one with this line of dialogue.
```suggestion
			`	"Well, come on boy. Let's go."`
			`	Caesar eagerly follows you back to your ship. He seems greatly relieved to be with you, and not left all alone at the park.`
```
You could maybe use a dialogue tag, here, too.

> 			`	Once back on your ship, you make sure he's fed, watered, and settled in for his interstellar journey.`

Reviewer 4: ```suggestion
			`	Once back on the <ship>, you make sure he's fed, watered, and settled in for his interstellar journey.`
```

Reviewer 3: Actually your suggestion was the old version. But I then suggested to replace it by the current one.

Reviewer 7: I like `your`

Reviewer 4: Ah, okay then.

				accept

	on complete

> 		payment 1000

Author: lol Alex, that's the point.

			branch "transported pookie"
				has "Pookie, Part 2: done"
				not `outfit: "Puny"`

> 			`Caesar has been very obedient the whole trip, much to your convenience. He lays down by the airlock while your ship sets down in the local Avalonia spaceport, which is enclosed on all sides by a squalid urban landscape. The honeycombed homes of humble help. You rent a vehicle and take yourself and Caesar beyond the blighted city and out into the pleasant picture-perfect paradise of the wealthy. One princely estate after another rolls past you as you drive deeper into the rural domains of the Republic's super-wealthy. Eventually, you come across a large metal gate that reads, "Mitchell Ranch."`

Reviewer 4: "The honeycombed homes of humble help." isn't a full sentence.

Reviewer: It's a minor sentence. Not uncommon.

> 			`Caesar has been very obedient the whole trip, much to your convenience. He lays down by the airlock while your ship sets down in the local Avalonia spaceport, which is enclosed on all sides by a squalid urban landscape. The honeycombed homes of humble help. You rent a vehicle and take yourself and Caesar beyond the blighted city and out into the pleasant picture-perfect paradise of the wealthy. One princely estate after another rolls past you as you drive deeper into the rural domains of the Republic's super-wealthy. Eventually, you come across a large metal gate that reads "Mitchell Ranch."`

Reviewer 4: ```suggestion
			`Caesar has been very obedient the whole trip, much to your convenience. He lays down by the airlock while your ship sets down in the local Avalonia spaceport, which is enclosed on all sides by a squalid urban landscape. You rent a vehicle and take yourself and Caesar beyond the blighted city and out into the pleasant picture-perfect paradise of the wealthy. One princely estate after another rolls past you as you drive deeper into the rural domains of the Republic's super-wealthy. Eventually, you come across a large metal gate that reads "Mitchell Ranch."`
```

Or make it a sentence

				goto ranch

			label "transported pookie"

> 			`Caesar has been very obedient the whole trip, unlike some other certain dogs you've transported in the past. He lays down by the airlock while your ship sets down in the local Avalonia spaceport, which is enclosed on all sides by a squalid urban landscape. The honeycombed homes of humble help. You rent a vehicle and take yourself and Caesar beyond the blighted city and out into the pleasant picture-perfect paradise of the wealthy. One princely estate after another rolls past you as you drive deeper into the rural domains of the Republic's super-wealthy. Eventually, you come across a large metal gate that reads, "Mitchell Ranch."`

Reviewer 4: ```suggestion
			`Caesar has been very obedient the whole trip, unlike certain other dogs you've transported in the past. He lays down by the airlock while your ship sets down in the local Avalonia spaceport, which is enclosed on all sides by a squalid urban landscape. You rent a vehicle and take yourself and Caesar beyond the blighted city and out into the pleasant picture-perfect paradise of the wealthy. One princely estate after another rolls past you as you drive deeper into the rural domains of the Republic's super-wealthy. Eventually, you come across a large metal gate that reads, "Mitchell Ranch."`
```

			label ranch

> 			`	It's another 45 minutes of driving down long winding roads, past various out buildings, lodges, retreats, guest houses, and hobby barns, before at long last you arrive at the massive central mansion. Waiting at the bottom of stone stairs is a footman who opens the door for you and summons the senior butler.`

Reviewer 4: ```suggestion
			`	It's another forty-five minutes of driving down long winding roads, past various outbuildings, lodges, retreats, guest houses, and hobby barns, before at long last you arrive at the massive central mansion. Waiting at the bottom of stone stairs is a footman who opens the door for you and summons the senior butler.`
```

> 			`	It's another 45 minutes of driving down long winding roads, past various out buildings, lodges, retreats, guest houses, and hobby barns, before at long last you arrive at the massive central mansion. Waiting at the bottom of stone stairs is a footman who opens the door for you and summons the senior butler.`

Reviewer 7: Being a ranch doesn't feel right

- "massive central mansion" with footmen and butlers isn't what comes to mind when I think range.
- the demeanor of the woman doesn't feel like a ranch owner either

A 45 minute drive seems a bit long if we are already past the gate of the property.

Reviewer: It is called a ranch, but it's not really one. It's probably a name the Mitchells thought was exotic; it's still a country estate at heart.

Reviewer 7: The dog's name is "Augustus Caesar" which is a reference to the roman empire so my mind is preconditioned to envision some sort of massive stone type of architecture and that fits with the context and descriptions of this mission story. 'Mitchells Ranch' doesn't work. If it's not a ranch, why call it one?  Maybe, use Mitchells Villa or something similar. 
 But, it's just my opinion.

Reviewer 4: I think it's really that the Mitchells have chosen to call it "Mitchell Ranch." It isn't a ranch, but they can call it whatever they want.

Author: Filthy rich boomers cosplaying Yellowstone.

> 			`	It's another forty-five minutes of driving down long winding roads, past various outbuildings, lodges, retreats, guest houses, and hobby barns, before at long last you arrive at the massive central mansion. Waiting at the bottom of stone stairs is a footman who opens the door for you and summons the senior butler.`

Reviewer 4: I feel like you could probably split the first sentence here into two. Six commas is a bit much.

> 			`	The butler arrives and looks you over unhappily. "The lady of the manor is not expecting... guests. Perhaps, you've misread your appointment, or more likely, arrived at the wrong estate."`

Reviewer 4: This reads a little awkwardly, with the "or more likely" not being separated by a comma. I'm not sure how you'd fix it, though, since adding a comma there would make the sentence very choppy.

			choice
				`	(Show the butler Caesar, and ask to see his owner.)`

> 			`	The butler's frown deepens, and then sighs. "Card, please." He holds out a silver tray and awaits your calling card.`

Author: If this is a common part of the ultra-rich culture it may be something ubiquitous enough in your society to be aware of.

> 			`	The butler's frown deepens, and then sighs. "Card, please." He holds out a silver tray and awaits your calling card.`

Reviewer 6: Fair enough 👍

			choice

> 				`	(Explain you don't have a calling card.)`

Author: Who knows how many years its been since then. Plus maybe you were a big fan of space downton abby? Or it's a more ubiquitous part of their culture than it is ours.

> 				`	(Explain you don't have a calling card.)`

Reviewer 6: I think it's part of the intro, so I assume you quit your job right before the start date.

> 				`	(Explain you don't have a calling card.)`

Author: Sorry, I was editing my response above when you replied. So they may not mesh anymore.

					goto nocard

> 				`	(Place your card on the tray.)`

Author: I wanted there to be an option for a pilot to have their own calling card. I don't know who they are or what there past was like and didn't want to assume they did not.

> 				`	(Place your card on the tray.)`

Reviewer 5: Perhaps we could gate this option behind having a certain net worth?
```suggestion
				`	(Place your card on the tray.)`
					to display 
						"net worth" > 20000000
```

			`	A quarter of an hour later a woman wearing an exceedingly impractical dress arrives.`

			label boolinbrook
			`	"Mr. Boolinbrook says you've brought back my darling See-See?" From down her nose she looks at you and waits for your response.`
			choice
				`	"Your dog is safe and sound with me."`
				`	"I found Caesar alone, out in a park on Calda."`
				`	"Here he is, you're welcome."`

			`	Before you can respond, Caesar starts yapping happily and runs up to the lady.`
			`	She snatches up the dog and draws it to her bosom. "Oh my darling See-See! It has been most dreadful waiting for news of you!"`
			`	After a moment she turns her attention back to you, looking slightly softer than before. "See-See tires of the parks here, and so every Tuesday I send her to Calda for a change of pace. It's good for her constitution, you know. Anyways, the horrid man who normally ferries our dogs on these little excursions had some sort of episode. A stroke, or heart-attack, you get the idea. He had an assistant return the dogs, but in the confusion they forgot my little See-See. I was sure some gauche tourist would have snatched her up and eaten her by now."`

> 			`	She turns her attention back to the dog, and speaks in a baby-voice, "But, instead, this gallant captain saved you!"`

Reviewer 6: Nah, "baby talk" is the way you speak to babies or pets. You would use a higher pitched voice and slur your words a bit in a cutesy way. I've never heard anyone use the term "baby voice," except to refer to singers with high pitched voices, like Grimes or Joanna Newsom.

			`	She procures a credit chip from somewhere in her gown and passes it to you. "I realize this is probably a fortune to you. I recommend you find a financial advisor and invest it well."`
			`	With that she turns and leaves; the butler likewise disappears, and you're escorted by the footman back to your vehicle.`

> 			`	You look down at the credit chip in your hand and read its value; 1,000 credits.`

Reviewer 4: ```suggestion
			`	You look down at the credit chip in your hand and read its value; <payment>.`
```

> 			`	You look down at the credit chip in your hand and read its value; 1,000 credits.`

Reviewer 7: I commend you on what you have written. I don't have the ability to write stories but I can spot when things don't feel right. 

This whole section feels confusing
-  the butler doesn't recognize the dog but later we are told that butler informed 'the lady of the manor' that you had returned her dog
- the waiting 'time' is too long for somebody that was very upset about her "darling See-See" 
- choices
  - "Your dog is safe and sound with me."     Can't she see the dog?
  - "I found Caesar alone out on Calda, by the spaceport."     No, you where in a park a considerable distance walk away from the spaceport.
  - You offer choices but then start the next dialogue with "Before you can respond" 
- "See-See tires of the parks here, ...." Doesn't the dog live on a huge estate that takes at least 45 minutes to reach the mansion?
- "But, instead, this **_gallant captain_** saved you!"  but in the next breath she calls you a vagabond and treats you with zero respect.

Reviewer: Partly, these are explained by the fact that rich Hestians (or rich Paradise Worlders) seem quite irrational to the player; they live on a different plane of existence to the player character (no matter how rich the player character is). They don't trust outsiders; they have these romanticised or incorrect notions about who ship captains are, how much money they actually make at all, and what you actually did. A lot of their demeanour is very much extraneous and shallow; taking 15 minutes to pick up a dog they supposedly love? Maybe they don't love the dog. Maybe that's why they send the dog away to Calda every Tuesday. They have this huge estate with hundreds of buildings, but they might not necessarily use them, or allow people and dogs to go roaming around, for whatever reason.

> "I found Caesar alone out on Calda, by the spaceport." No, you where in a park a considerable distance walk away from the spaceport.

Fair point though.

> You offer choices but then start the next dialogue with "Before you can respond"

It's not unprecedented, you're put in a similar situation in the Free Worlds campaign. (The more practical reason... it's sometimes easier to put the player in this situation than to write all of that.)

Author: Bene has the right of it above.
The disconnect is that this person is absurdly wealthy. It takes 45 minutes to get to their home because they live on a 100,000 acre practical fiefdom. As for it being a ranch, it's quite common for very wealthy people to own ranches, or even hobby farms they don't work on, or do very little on. Maybe they 'train' the horses, or work on 'dressage' for the Olympics. Those same people will of course also have servants. 

Same goes for her imagining her dog needs to be taken to a different planet because her 'humble yard' bores him. Not all wealthy people behave this way, but many certainly do. Dog's getting rides in private jets etc, taken to private beaches. 

As for the gallant captain vs vagabond phrasing this has the same reasoning as above. The same way a wealthy person could say they could never live without their beloved maid/butler/nurse/aid etc but then also be dismissive of them as human beings, or at least other people like them (poors? immigrants?). She's inconsistent because she doesn't view or understand you as a real person.

mission "Lost Dog: Homeward Bound"
	to offer
		random < 3
		has "Lost Dog: declined"
	source
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"

> 	destination "Hestia"

Reviewer 4: This seems unnecessary?

Reviewer 3: Indeed.

	on offer
		conversation

> 			`As you're walking through the spaceport your eye is drawn to a marginally familiar figure on a nearby newsfeed. You can't quite make out exactly what it is saying over the din, but you're able to gather that little lost dog you ran into on Calda, Caesar, has become something of a minor celebrity after managing to get back home to Hestia all by himself. The newsfeed scrolls down to show images of Caesar skiing, sneaking on transports, and riding automatic farming equipment.`

Reviewer 4: ```suggestion
			`As you're walking through the spaceport, your eye is drawn to a marginally familiar figure on a nearby newsfeed. You can't quite make out exactly what it is saying over the din, but you're able to gather that little lost dog you ran into on Calda, Caesar, has become something of a minor celebrity after managing to get back home to Hestia all by himself. The newsfeed scrolls down to show images of Caesar skiing, sneaking on transports, and riding automatic farming equipment.`
```

			`	As you walk, you eventually come across a perfectly manicured garden park named "Parc Manifold." You walk toward the gates to discover a ticket office, with signs advertising a free entry promotion this week only, crossing out the 3,400 credit regular admission fee.`
			choice
				`	(Go to the park.)`
					goto park
				`	(Head back to your ship.)`

			`	You have more important things to do than wander around an expensive terraformed garden.`
				decline

			label park
			`	The park has been meticulously sculpted so that no matter where you stand, it appears as though you are in the center of a lush naturalist painting. Colorful, comforting flora bleed away from you at all angles, and adorable yet inaccessible fauna seem to hug the very periphery of your every step.`
			`	The air is clear and crisp, and even the crunch of leaves and thud of the earth beneath your feet seems somehow delicious and satisfying. Finally, the sun begins to set and it is time to head back.`
			`	As you approach the exit to the park a small dog pads over to you and looks at you with mournful eyes. It seems to be alone, with no owner anywhere in sight.`

> 			`	"Come here," you call to the dog, which looks to be a poodle mix, and it obediently does as you command. You see that the dog is a male and check his collar:`

Author: Done.

			`	She turns her attention back to the dog, and speaks in baby talk, "But, instead, this gallant captain saved you!"`
			`	The lady snaps her fingers, and the butler is immediately at her side. "We must make this person whole. I can't be perceived as owing favors to vagabonds. Imagine the talk!`
			`	The butler quickly procures a credit chip and shows it to her, accepting a curt nod for approval. "I realize that this is probably a fortune to you," she says. "I recommend you find a financial advisor and invest it well."`
			`	With that, she turns and leaves; the butler likewise disappears, and you're escorted by the footman back to your vehicle.`
			`	You look down at the credit chip in your hand and read its value: <payment>.`

mission "Lost Dog: Homeward Bound"
	to offer
		random < 3
		has "Lost Dog: declined"
	source
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
	on offer
		conversation

> 			`As you're walking through the spaceport, your eye is drawn to a marginally familiar figure on a nearby newsfeed. Although you can't make out exactly what it is saying over the din, you are able to gather that the little lost dog you ran into on Calda has become something of a minor celebrity after it managed to get back home to Hestia all by itself. The newsfeed scrolls down to show images of the dog - whose name is Caesar - skiing, sneaking on transports, and riding automatic farming equipment.`

Reviewer 5: ```suggestion
			`As you're walking through the spaceport, your eye is drawn to a marginally familiar figure on a nearby newsfeed. Although you can't make out exactly what it is saying over the din, you are able to gather that the little lost dog you ran into on Calda has become something of a minor celebrity after it managed to get back home to Glory all by itself. The newsfeed scrolls down to show images of the dog - whose name is Caesar - skiing, sneaking on transports, and riding automatic farming equipment.`
```

			`	She turns her attention back to the dog, and speaks in baby talk, "But, instead, this gallant captain saved you!"`
			`	The lady snaps her fingers, and the butler is immediately at her side. "We must make this person whole. I can't be perceived as owing favors to vagabonds. Imagine the talk!`
			`	The butler quickly procures a credit chip and shows it to her, accepting a curt nod for approval. "I realize that this is probably a fortune to you," she says. "I recommend you find a financial advisor and invest it well."`
			`	With that, she turns and leaves; the butler likewise disappears, and you're escorted by the footman back to your vehicle.`
			`	You look down at the credit chip in your hand and read its value: <payment>.`

mission "Lost Dog: Homeward Bound"
	to offer
		random < 3
		has "Lost Dog: declined"
	source
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
	on offer
		conversation

> 			`As you're walking through the spaceport, your eye is drawn to a marginally familiar figure on a nearby newsfeed. Although you can't make out exactly what it is saying over the din, you are able to gather that the little lost dog you ran into on Calda has become something of a minor celebrity after it managed to get back home to Glory all by itself. The newsfeed scrolls down to show images of the dog - whose name is Caesar - skiing, sneaking on transports, and riding automatic farming equipment.`

Reviewer 5: ```suggestion
			`As you're walking through the spaceport, your eye is drawn to a marginally familiar figure on a nearby newsfeed. Although you can't make out exactly what it is saying over the din, you are able to gather that Caesar, the little lost dog you ran into on Calda, has become something of a minor celebrity after it managed to get back home to Glory all by itself. The newsfeed scrolls down to show images of the dog skiing, sneaking on transports, and riding automatic farming equipment.`
```

			`As you are walking around the spaceport, you see that more than a few pilots are watching the news on nearby displays. There are mixed emotions amongst the crowd - some pilots are having a good laugh while others seem disappointed or even confused. You aren't sure what this is about until, lo and behold, the result of that Hogshead naming poll from a while ago is clearly shown on a display - Shippy McShipface. You can't say it's all that much worse than some of the names you've heard pirates give their ships before, but it still seems silly that the transport company is actually going through with the name.`
			`	All of the hyped discussions around Hogsheads in the past gave them a brief but strong popularity amongst transport organizations looking to grab the average passenger's attention. In the end, the company responsible for this naming poll got what they wanted, and a little bit of the Hogshead's spirit has been brought back, along with the interest of several pilots. Perhaps the name "Shippy McShipface" is fitting after all.`
				decline

mission "Lost Dog"

> 	name "Bring a dog back to <planet>"

Reviewer 3: ```suggestion
	name "Bring a lost dog back to <planet>"
```

> 	name "Bring a dog back to <planet>"

Reviewer 3: ```suggestion
	name "Bring a dog back that've been lost to <destination>."
```

Reviewer: The field for the name in the mission panel is quite small, so we have to be careful about how much we put in there.

	description "Bring Caesar, a dog you found lost on <source>, to their owner on <destination>."
	to offer
		random < 15
	source "Calda"
	destination "Hestia"
	on offer
		conversation

> 			`Your walk from the <ship> takes you outside <planet>'s bustling spaceport and into the overtly welcoming and carefully inoffensive tourist-saturated streets of the spaceport town. Every path seems purpose-built to whisk you further away from your ship and deeper into the unnatural radiance of this Paradise World. The hand-crafted beauty is inescapable, a never-ending sequence of breathtaking views.`

Reviewer 3: ```suggestion
			`Your walk from your ship <ship>, take you outside <planet>'s bustling spaceport and into the overtly welcoming and carefully inoffensive tourist-saturated streets of the spaceport town. Every path seems purpose-built to whisk you further away from your ship and deeper into the unnatural radiance of this Paradise World. The hand-crafted beauty is inescapable, a never-ending sequence of breathtaking views.`
```

I think this syntax's better, in my opinion.

Reviewer: Personally I find this a bit wordy.

Reviewer 7: First OP version reads better.

Author: How about.. "The walk from your ship takes you outside..."

			`	As you walk you eventually come across an immaculate and perfectly manicured garden park. Normally there is an exorbitant fee to enter, but due to some promotion today you can enter freely.`
			choice
				`	(Go to the park.)`
					goto park

> 				`	(Head back to the ship.)`

Reviewer 3: ```suggestion
				`	(Head back to your ship.)`
```

			`	It was a splendid walk, but you decide it's time to head back. You have more important things to do than wander around an expensive terraformed garden.`
				decline

			label park
			`	The park has been meticulously sculpted so that no matter where you stand it appears as though you are in the center of a lush naturalist painting. Colorful, comforting flora bleed away from you at all angles, and adorable yet inaccessible fauna seem to hug the very periphery of your every step.`
			`	The air is clear and crisp, and even the crunch and thud of the ground beneath your feet seem somehow delicious and satisfying. Finally, the sun begins and set and it is time to head back.`
			`	As you approach the exit to the park a small dog pads over to you and looks at you with morunful eyes. The dog seems to be alone with no owner anywhere in sight.`

> 			`	"Come here." You call to the poodle-hybrid, who obediently does as you command. You see that its a male and check its collar;`

Reviewer 3: I agree with that.

			``

> 			`	"Caesar Augustus - Property of the Mitchell's"`

Reviewer 3: I agree with that.

			`	3 Mitchell Drive`
			`	Avalonia District, Hestia`
			``

> 			`	"That's odd." You say to Caesar. "How did you get lost on the wrong planet, huh boy?"`

Reviewer: ```suggestion
			`	"That's odd," you say to Caesar. "How did you get lost on the wrong planet, huh boy?"`
```

			`	You pet Caesar, who seems glad of the attention, and decide what to do.`
			choice
				`	(Take Caesar to his owners on Hestia.)`
					goto takehim
				`	(Leave Caesar here.)`

> 			`	Perhaps his owners are on holiday and will be back looking for him any minute. Although it's getting dark and no one is around, you decide the best course of action to leave Caesar at the park.`

Reviewer 3: ```suggestion
			`	Perhaps his owners are on vacation and will be back looking for him any minute. Although it's getting dark and no one seems to be around, you decide the best course of action to leave Caesar at the park.`
```
Since the game's spelled the american way, it's logical to replace holiday by vacation.

> 			`	As you walk off you hear Caesar whining, and as you look back you notice him slink off into the park. But you've made your decision; hopefully everything works out for the best.`

Reviewer 3: ```suggestion
			`	As you walk off you hear Caesar whining, and as you look back you notice him slink off into the park. But you've made your decision; hopefully everything will out for the best.`
```

I'm not sure about that though.

				decline

			label takehim
			`	"Well, come on boy. Let's go." Caesar eagerly follows you back to your ship. He seems greatly relieved to be with you, and not left all alone at the park.`

> 			`	Once on board your ship, you make sure he's fed, watered, and settled in for the interstellar journey.`

Reviewer 3: ```suggestion
			`	Once back on your ship, you make sure he's fed, watered, and settled in for his interstellar journey.`
```

	on complete
		payment 1000
		conversation

> 			`Your ship sets down in a squalid urban landscape. The honeycombed homes of humble help. You rent a vehicle and take yourself and Caesar beyond the blighted city and out into the pleasant picture-perfect paradise of the wealthy. One princely estate after another rolls past you as you drive deeper into the rural domains of the Republic's super-wealthy. Eventually, you come across a large metal gate that reads, "Mitchell Ranch."`

Reviewer 3: ```suggestion
			`Your ship carefully lands in a squalid urban landscape. The honeycombed homes of humble help. You rent a vehicle and take yourself and Caesar beyond the blighted city and out into the pleasant picture-perfect paradise of the wealthy. One princely estate after another rolls past you as you drive deeper into the rural domains of the Republic's super-wealthy. Eventually, you come across a large metal gate that reads, "Mitchell Ranch."`
```

Reviewer: Might consider rewriting this to include a spaceport, right now it sounds like you've parked your ship somewhere ad-hoc.

Reviewer: Okay, I've added a mention of the spaceport, might be pending further changes depending on how SpearDane would like to write it.

Reviewer 7: "squalid urban landscape" doesn't really fit the descriptions of Hestia.

Reviewer: Hestia's description has something to this effect:
`In Hestia, the poor live in cities; the rich live in country estates.`
I imagine that the cities might be at least a little run-down.

			`	She turns her attention back to the dog, and speaks in a baby-voice, "But, instead, this gallant captain saved you!"`
			`	The lady snaps her fingers, and the butler is immediately at her side. "We must make this person whole. I can't be owing favors to vagabonds. Imagine the talk!`
			`	She procures a credit chip from somewhere in her gown and passes it to you. "I realize this is probably a fortune to you, I recommend you find a financial advisor and invest it well."`
			`	With that she turns and leaves; the butler likewise disappears, and you're escorted by the footman back to your vehicle.`
			`	You look down at the credit chip she's given you and read its value; 1,000 credits.`

> 				decline

Reviewer 3: ```suggestion
```

It's a completion conversation: why declining?
